src/components/main/Main.tsx

```diff
@@ -0,0 +1,77 @@
+import { Todo } from '../../types/Todo';
+import {
+  deleteTodo,
+  updateTodoCompleted,
+  // updateTodoTitle,
+} from '../../api/todos';
+
+interface Props {
+  todos: Todo[] | null;
```

Comment:
handle it in the upper component
```suggestion
  todos: Todo[];
```

Reply:
its gonna be Todo[] or [] 

handled it that way. pls lmk if you had different idea